utils/build-script

```diff
@@ -415,7 +415,7 @@ class BuildScriptInvocation(object):
         # Unset environment variables that might affect how tools behave.
         for v in [
                 'MAKEFLAGS',
-                'SDKROOT',
```

Comment:
Just a drive by: Why comment this out? Why not just delete it?

Reply:
No reason, just forgot to actually delete it before I committed.